### ISSUE TITLE
what's the use of 'text' in the model ?

### ISSUE THREAD

**Issue by gino07172002** [JaidedAI/EasyOCR#475]
hi sir, 
     I'm tracing your code to learn pytorch. 

     I find that in the "vgg_model.py",  There is a parameter called "text", it never used in the forward() function, however, it also appear in the "recognizer_predict" function to receive the  "image_tensors" variable.
    I find that even if I remove the "text" parameter in the  "vgg_model.py" and make the "recognizer_predict" function as

`preds = model(image)`

    the final output result is not changed. 
  
    Is the 'text' parameter in "vgg_model.py" redundant ?

**Comment by SamSamhuns** [comment]
Yes, it looks like the `text` parameter is not used anywhere in this repo. Might be a relic of some testing from this code in the repo deep-text-recognition-benchmark https://github.com/clovaai/deep-text-recognition-benchmark/blob/master/modules/prediction.py

**Comment by Ismail-Maj** [comment]
It could be useful later if the model changes from CTC to attention
